runzero-api.yml

```diff
@@ -18,13 +18,18 @@ tags:
     description: Requires Export Token or Organization Key
   - name: Public
     description: Unauthenticated
+    
+security:
+  - bearerAuth: []
+  - oauthDefaults: []
```

Comment:
this sets the default security to API Key / OAuth, explicitly setting no security where needed